package.json

```diff
@@ -20,19 +20,24 @@
     "dev": "rslib build --watch",
     "prebundle": "node ./bin.js",
     "prepare": "npm run build",
-    "bump": "npx bumpp"
+    "bump": "npx bumpp",
+    "test": "pnpm build && rstest"
```

Comment:
Is `pnpm build` required here? The `prepare` script should be called after installation.